bugbug/model.py

```diff
@@ -192,19 +192,45 @@ def train(self, importance_cutoff=0.15):
             explainer = shap.TreeExplainer(self.clf)
             shap_values = explainer.shap_values(X_train)
 
-            # TODO: Actually implement feature importance visualization for multiclass problems.
+            avg_shap_values = 0
             if isinstance(shap_values, list):
-                shap_values = np.sum(np.abs(shap_values), axis=0)
+                avg_shap_values = np.sum(np.abs(shap_values), axis=0)
+            else:
+                avg_shap_values = shap_values
 
             important_features = self.get_important_features(
-                importance_cutoff, shap_values
+                importance_cutoff, avg_shap_values
             )
 
-            print(f"\nTop {len(important_features)} Features:")
-            for i, [importance, index, is_positive] in enumerate(important_features):
-                print(
-                    f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})'
-                )
+            if not isinstance(shap_values, list):
```

Comment:
I'd suggest refactoring this code to use tabulate, this would make it much cleaner.

Reply:
you mean, same as multi class make it  horizontal instead of vertical ?

Reply:
I meant using `tabulate`, which is a Python package to pretty-print tabular data such as this (both in the binary and multi-class cases).